src/index.js

```diff
@@ -89,14 +97,22 @@ class Dropzone extends React.Component {
   onDragEnter(evt) {
     evt.preventDefault()
 
+    const items = getDataTransferItems(evt)
```

Comment:
`getDataTransferItems` is `async` now so I'm not sure it's going to work like this. Are tests passing?

Reply:
Hmm, I'm just trying to figure that out, but the only possibility is to wait for the promise to resolve before doing anything in these `onDrag*` callbacks. Is there any reason why this is now async?